components/images/gallery.tsx

```diff
@@ -55,17 +57,41 @@ export default function Gallery({ longitude, latitude }: GalleryProps) {
   }, []);
 
   if (!images || !images.length) {
-    return <Heading>Pas d'images disponibles</Heading>;
+    return (
+      <>
+        <HStack height="100%" alignItems="center">
+          <VStack width="100%" alignItems="center">
+            <Text fontSize="3xl">Pas d'images disponibles</Text>
+          </VStack>
+        </HStack>
+        <Fab
+          placement="top-left"
+          bgColor="#19837C"
+          icon={<Icon color="white" as={AntDesign} name="caretleft" size="4" />}
+          onPress={() => router.back()}
+          renderInPortal={false}
+        />
+      </>
+    );
   }
 
   return (
-    <ScrollView>
-      {images?.map((image: { file: string; name: string }, i: number) => (
-        <VStack key={i.toString()} alignItems="center" mb="5">
-          <Image source={{ uri: image.file }} alt={i.toString()} w="650" h="300" mb="3" />
-          <Text fontSize="xl">{image.name}</Text>
-        </VStack>
-      ))}
-    </ScrollView>
+    <>
+      <ScrollView>
+        {images?.map((image: { file: string; name: string }, i: number) => (
+          <VStack key={i.toString()} alignItems="center" mb="5">
```

Comment:
L'existant est ok mais c'est juste une préférence perso:
```suggestion
          <VStack key={`${i}`} alignItems="center" mb="5">
```